index.ts

```diff
@@ -0,0 +1,141 @@
+import { FastifyPluginAsync } from "fastify";
+import { Actor, Item, Member, PostHookHandlerType, PreHookHandlerType, UnknownExtra } from "graasp";
+
+interface GraaspS3FileItemOptions {
+  type: string;
```

Comment:
you can add a comment over each option. it will popup as documentation to the developer using the plugin, and settings the options:
```
interface GraaspFileUploadLimiterOptions {
  /** Item type to target (ex: 'file', 's3File') */
  type: string;
  /** Chain of property names to the file size (ex: 'extra.s3File.size')  */
  sizePath: string;
}
````